### PR TITLE
[FIX](serde )pick 27965 for decimalv2

### PR DESCRIPTION
**Pull request by amorynan**
## Proposed changes
fix datetimev2 serde parse from string with scale
pick: https://github.com/apache/doris/pull/27965
Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by amorynan** [comment]
run buildall

**Comment by amorynan** [comment]
run p0

File: be/src/vec/data_types/serde/data_type_datetimev2_serde.cpp [hunk 1]
```diff
@@ -79,8 +79,8 @@ Status DataTypeDateTimeV2SerDe::deserialize_one_cell_from_json(IColumn& column,
         }
 
     } else if (ReadBuffer rb(slice.data, slice.size);
-               !read_datetime_v2_text_impl<UInt64>(val, rb)) {
-        return Status::InvalidDataFormat("parse date fail, string: '{}'",
+               !read_datetime_v2_text_impl<UInt64>(val, rb, scale)) {
+        return Status::InvalidArgument("parse date fail, string: '{}'",
                                          std::string(rb.position(), rb.count()).c_str());
```
Comment (yiguolei) [inline be/src/vec/data_types/serde/data_type_datetimev2_serde.cpp:84]: why not pick related regression tests?

Comment (amorynan) [reply be/src/vec/data_types/serde/data_type_datetimev2_serde.cpp:84]: the out file is generate from master which round behavior is not compatible for 2.0

**Comment by amorynan** [comment]
run buildall

**Pull request merged**